web/docs/tutorials/todo-app/07-dependencies.md

```diff
@@ -50,8 +54,40 @@ const Clocks = () => {
 
 export default Clocks
 ```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```tsx title="src/client/Clocks.tsx"
+import { useEffect, useState } from 'react'
+import Clock from 'react-clock'
```

Comment:
Error (in the editor only, works fine in `wasp start`) about missing `.d.ts` file for `react-clock`. I assume we need to add that to dependencies in main.wasp too?

Reply:
Updating to the latest version seems to help :)